{"uid":"a5078533f69a5c50","name":"Проверить отображения проектов ожидающих модерации","fullName":"tests.test_profile.TestProfile#test_displays_projects_awaiting_moderation","historyId":"c929eaafb49f01d6cb13c05b2782e17d","time":{"start":1738865303812,"stop":1738865303812,"duration":0},"description":"Проверяется, что при открытие профиля отображаются проекты ожидающих модерации в количестве > 0","descriptionHtml":"<p>Проверяется, что при открытие профиля отображаются проекты ожидающих модерации в количестве &gt; 0</p>\n","status":"broken","statusMessage":"KeyError: 'access_token'","statusTrace":"request = <SubRequest 'login_to_app' for <Function test_displays_projects_awaiting_moderation>>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"427b8e5c169331b20cbdf7577779cc30\")>\n\n    @pytest.fixture(scope='function')\n    @allure.title('Авторизация пользователя в системе')\n    @allure.step('Авторизация пользователя в системе')\n    def login_to_app(request, driver):\n        \"\"\" Фикстура Логина в систему \"\"\"\n        login_page = ApiAuthorization(driver)\n>       login_page.user(User.LOGIN, User.PASSWORD)\n\nconftest.py:53: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <model.api.authorization.ApiAuthorization object at 0x7f0ee39767b0>\nusername = 'Aleska', password = 'qwe123'\n\n    def user(self, username: str, password: str) -> None:\n        \"\"\"\n        Метод авторизации пользователя по средствам API.\n    \n        :param username: Логин пользователя\n        :param password: Пароль пользователя\n        \"\"\"\n        headers = {\n            'X-Client-Id': 'round',\n            'X-Client-Type': 'web',\n            'X-Language': 'ru'\n        }\n        data = {\n            'client_id': 'Krujok.Online.Mobile',\n            'grant_type': 'password',\n            'username': username,\n            'password': password\n        }\n    \n        response = requests.post(url=Api.TOKEN, headers=headers, data=data)\n>       access_token = response.json()[\"access_token\"]\nE       KeyError: 'access_token'\n\nmodel/api/authorization.py:30: KeyError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1738864340794,"stop":1738864340795,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"driver","time":{"start":1738865303812,"stop":1738865304224,"duration":412},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"setup","time":{"start":1738865304224,"stop":1738865304225,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Инициализация локаторов","time":{"start":1738865304225,"stop":1738865304225,"duration":0},"status":"passed","steps":[{"name":"Инициализация локаторов","time":{"start":1738865304225,"stop":1738865304225,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"request","value":"<SubRequest 'elements' for <Function test_displays_projects_awaiting_moderation>>"}],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":1,"hasContent":true,"attachmentStep":false},{"name":"Авторизация пользователя в системе","time":{"start":1738865304225,"stop":1738865304860,"duration":635},"status":"broken","statusMessage":"KeyError: 'access_token'\n","statusTrace":"  File \"/usr/local/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/usr/local/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/_pytest/fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/usr/workspace/conftest.py\", line 53, in login_to_app\n    login_page.user(User.LOGIN, User.PASSWORD)\n  File \"/usr/workspace/model/api/authorization.py\", line 30, in user\n    access_token = response.json()[\"access_token\"]\n                   ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^\n","steps":[{"name":"Авторизация пользователя в системе","time":{"start":1738865304225,"stop":1738865304860,"duration":635},"status":"broken","statusMessage":"KeyError: 'access_token'\n","statusTrace":"  File \"/usr/local/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/usr/workspace/conftest.py\", line 53, in login_to_app\n    login_page.user(User.LOGIN, User.PASSWORD)\n  File \"/usr/workspace/model/api/authorization.py\", line 30, in user\n    access_token = response.json()[\"access_token\"]\n                   ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^\n","steps":[],"attachments":[],"parameters":[{"name":"request","value":"<SubRequest 'login_to_app' for <Function test_displays_projects_awaiting_moderation>>"},{"name":"driver","value":"<selenium.webdriver.chrome.webdriver.WebDriver (session=\"427b8e5c169331b20cbdf7577779cc30\")>"}],"shouldDisplayMessage":true,"attachmentsCount":0,"stepsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":1,"hasContent":true,"attachmentStep":false}],"afterStages":[{"name":"driver::0","time":{"start":1738865304866,"stop":1738865305044,"duration":178},"status":"passed","steps":[],"attachments":[{"uid":"4c28c78618af13a6","name":"page_source","source":"4c28c78618af13a6.html","type":"text/html","size":39},{"uid":"d3f7cb21fa23f61b","name":"screenshot","source":"d3f7cb21fa23f61b.png","type":"image/png","size":10592},{"uid":"ab7d246816cf4ef1","name":"browser_logs","source":"ab7d246816cf4ef1.txt","type":"text/plain","size":0}],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":3,"stepsCount":0,"hasContent":true,"attachmentStep":false}],"labels":[{"name":"feature","value":"Профиль пользователя"},{"name":"severity","value":"Critical"},{"name":"tag","value":"regression"},{"name":"tag","value":"profile"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_profile"},{"name":"subSuite","value":"TestProfile"},{"name":"host","value":"6af98febc332"},{"name":"thread","value":"1-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_profile"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"critical","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":17,"skipped":0,"passed":27,"unknown":0,"total":44},"items":[{"uid":"6c76297a776ba915","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: Не найдены элементы с локатором ('xpath', '//div[@data-testing=\"project_moderation\"]')","time":{"start":1738269221329,"stop":1738269251501,"duration":30172}},{"uid":"6e665bcdad5959c0","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: Не найдены элементы с локатором ('xpath', '//div[@data-testing=\"project_moderation\"]')","time":{"start":1738099664141,"stop":1738099694305,"duration":30164}},{"uid":"f4d16c62ae6fcb02","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: Не найдены элементы с локатором ('xpath', '//div[@data-testing=\"project_moderation\"]')","time":{"start":1738005544694,"stop":1738005574864,"duration":30170}},{"uid":"f2de17ee4473786a","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: Не найдены элементы с локатором ('xpath', '//div[@data-testing=\"project_moderation\"]')","time":{"start":1738002836626,"stop":1738002866787,"duration":30161}},{"uid":"2c0f32f5e364848a","status":"broken","statusDetails":"urllib3.exceptions.ReadTimeoutError: HTTPConnectionPool(host='localhost', port=36019): Read timed out. (read timeout=120)","time":{"start":1737931477590,"stop":1737931477590,"duration":0}},{"uid":"4cbd4b8b81ebf75e","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: Не найдены элементы с локатором ('xpath', '//div[@data-testing=\"project_moderation\"]')","time":{"start":1737843200932,"stop":1737843232648,"duration":31716}},{"uid":"68d397231d67e50","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: Не найдены элементы с локатором ('xpath', '//div[@data-testing=\"project_moderation\"]')","time":{"start":1737832398379,"stop":1737832430098,"duration":31719}},{"uid":"733d8956a3d49ea5","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: Не найдены элементы с локатором ('xpath', '//div[@data-testing=\"project_moderation\"]')","time":{"start":1737799759912,"stop":1737799791625,"duration":31713}},{"uid":"f6f3547fbecc984","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: Не найдены элементы с локатором ('xpath', '//div[@data-testing=\"project_moderation\"]')","time":{"start":1737757989591,"stop":1737758021303,"duration":31712}},{"uid":"bf7d8708ea1f8e65","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: Не найдены элементы с локатором ('xpath', '//div[@data-testing=\"project_moderation\"]')","time":{"start":1737744986845,"stop":1737745018581,"duration":31736}},{"uid":"ab2acea999725af","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: Не найдены элементы с локатором ('xpath', '//div[@data-testing=\"project_moderation\"]')","time":{"start":1737652869888,"stop":1737652901606,"duration":31718}},{"uid":"3cf5e13927a52e5","status":"passed","time":{"start":1737570552170,"stop":1737570554275,"duration":2105}},{"uid":"54848f8dcfa05b29","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/chromium-browser is no longer running, so ChromeDriver is assuming that Chrome has crashed.)","time":{"start":1737568967027,"stop":1737568967027,"duration":0}},{"uid":"db0c1a175a2f2a20","status":"passed","time":{"start":1734034837432,"stop":1734034839019,"duration":1587}},{"uid":"b8f460faf19b975c","status":"passed","time":{"start":1733937521177,"stop":1733937524323,"duration":3146}},{"uid":"d109b6ddd8aa1561","status":"passed","time":{"start":1733866096231,"stop":1733866097822,"duration":1591}},{"uid":"95da8635223bc03b","status":"passed","time":{"start":1733860543537,"stop":1733860545121,"duration":1584}},{"uid":"1dcff29dfeb03931","status":"passed","time":{"start":1733857095391,"stop":1733857097102,"duration":1711}},{"uid":"7445703071e629a8","status":"passed","time":{"start":1733771984077,"stop":1733771985662,"duration":1585}},{"uid":"1e6b7c9c1d16bbaf","status":"passed","time":{"start":1733766462043,"stop":1733766463629,"duration":1586}}]},"tags":["regression","profile"]},"source":"a5078533f69a5c50.json","parameterValues":[]}